hedera-mirror-test/src/test/java/com/hedera/mirror/test/e2e/acceptance/steps/EstimateFeature.java

```diff
@@ -0,0 +1,483 @@
+    private static final String DESTROY_SELECTOR = "83197ef0";
+    private static final int DESTROY_ACTUAL_GAS = 51193;
+    private static final String WRONG_METHOD_SIGNATURE_SELECTOR = "ffffffff";
+    private static final String CALL_TO_INVALID_CONTRACT_SELECTOR = "70079963";
+    private static final int CALL_TO_INVALID_CONTRACT_ACTUAL_GAS = 24374;
+    private static final String DELEGATE_CALL_TO_INVALID_CONTRACT_SELECTOR = "7df6ee27";
+    private static final int DELEGATE_CALL_TO_INVALID_CONTRACT_ACTUAL_GAS = 24350;
+    private static final String STATIC_CALL_TO_INVALID_CONTRACT_SELECTOR = "41f32f0c";
+    private static final int STATIC_CALL_TO_INVALID_CONTRACT_ACTUAL_GAS = 24394;
+    private static final String CALL_CODE_TO_INVALID_CONTRACT_SELECTOR = "e080b4aa";
+    private static final int CALL_CODE_TO_INVALID_CONTRACT_ACTUAL_GAS = 24031;
+    private static final String CALL_CODE_TO_EXTERNAL_CONTRACT_FUNCTION_SELECTOR = "4929af37";
+    private static final int CALL_CODE_TO_EXTERNAL_CONTRACT_FUNCTION_ACTUAL_GAS = 26100;
+    private static final String DELEGATE_CALL_CODE_TO_EXTERNAL_CONTRACT_FUNCTION_SELECTOR = "80f009b6";
+    private static final int DELEGATE_CALL_CODE_TO_EXTERNAL_CONTRACT_FUNCTION_ACTUAL_GAS = 24712;
+    private static final String CALL_CODE_TO_EXTERNAL_CONTRACT_FUNCTION_VIEW_SELECTOR = "fa5e414e";
+    private static final int CALL_CODE_TO_EXTERNAL_CONTRACT_VIEW_FUNCTION_ACTUAL_GAS = 22272;
+    private static final String STATE_UPDATE_OF_CONTRACT_SELECTOR = "5256b99d";
+    private static final int STATE_UPDATE_OF_CONTRACT_ACTUAL_GAS = 30500;
+    private static final String GET_GAS_LEFT_SELECTOR = "51be4eaa";
+    private static final int GET_GAS_LEFT_ACTUAL_GAS = 21326;
+    private static final String REENTRANCY_TRANSFER_ATTACK_SELECTOR = "ffaf0890";
+    private static final int REENTRANCY_TRANSFER_ATTACK_ACTUAL_GAS = 55500;
+    private static final String REENTRANCY_CALL_ATTACK_SELECTOR = "e7df080e";
+    private static final int REENTRANCY_CALL_ATTACK_ACTUAL_GAS = 55818;
+    private static final String NESTED_CALLS_SELECTOR = "bb376a96";
+    private static final int POSITIVE_NESTED_CALLS_ACTUAL_GAS = 45871;
+    private static final int LIMITED_NESTED_CALLS_ACTUAL_GAS = 525255;
+    private static final String HEX_DIGITS = "0123456789abcdef";
+    private static final String GET_ADDRESS_SELECTOR = "38cc4831";
+    private static final String INVALID_SELECTOR = "5a790dba";
+    private static final String GET_MOCK_ADDRESS_SELECTOR = "14a7862c";
+
+
+    private final ContractClient contractClient;
+    private final FileClient fileClient;
+    private final MirrorNodeClient mirrorClient;
+    private DeployedContract deployedContract;
+    private String contractSolidityAddress;
+    private int lowerDeviation;
+    private int upperDeviation;
+
+
+    @Value("classpath:solidity/artifacts/contracts/EstimateGasContract.sol/EstimateGasContract.json")
+    private Path estimateGasTestContract;
```

Comment:
This will fail to be loaded when the acceptance tests are ran in a container. Need to use as a `Resource` and get it as an InputStream. See `ERCContractFeature`.

```suggestion
    private Resource estimateGasTestContract;
```